docs/guides/hybrid-graphics.md

```diff
@@ -7,9 +7,36 @@ This has been tested on the MacBookPro16,1 and the MacBookPro15,1. The 15,3 and
 ## Issues
 
 1. Resume after suspend is broken, as the GMUX (graphics multiplexer) doesn't connect the iGPU to the display after resuming. For this to be fixed, a Linux driver for `acpi:APP000B:GPUC:` needs to be written (macOS uses AppleMuxControl2.kext). The extra battery life may make this a worthwhile trade-off (about 3 hours to almost 6 hours on a MacBookPro16,1)
+
 2. If using `DRI_PRIME=1` on programs causes system crashes, with "[CPU CATERR](https://gist.github.com/Redecorating/956a672e6922e285de83fdd7d9982e5e#gistcomment-3719941)" problem reports in macOS, disable dynamic power management with the `amdgpu.dpm=0` kernel argument, or `echo high | sudo tee /sys/bus/pci/drivers/amdgpu/0000:??:??.?/power_dpm_force_performance_level`.
+
 3. If apple-set-os is loaded, the iGPU will control display brightness, and if the iGPU isn't the boot gpu, the i915 Intel graphics driver will not load, and the display brightness cannot be changed (The exception to this is sometimes when rebooting from macOS Recovery etc, i915 loads fine).
 
+4. Some users with MacBookPro16,1 and AMD gpu have reported issues like AMD gpu uses too much power with excessive high temperatures under normal conditions or sudden high speed fan noise then instant system shut down, getting the T2 chip reset.
+
+    Posible workarounds:
+
+    1. Set iGPU as main gpu.
+
+    2. Set AMD gpu Dynamic Power Management from auto to low.
+
+    You can test it quickly with: `echo low | sudo tee /sys/class/drm/card0/device/power_dpm_force_performance_level`
+
+    To apply the low level permanently, create `etc/udev/rules.d/30-amdgpu-pm.rules` file with the following contents:
```

Comment:
**/**etc

Reply:
this is just an example and prefer to lead users to use low.

Reply:
all change, please have a look

Reply:
it seems like it's still a relative path instead of an absolute path (starting with "/"). I think this means it currently only will work if the current directory is /

Reply:
good catch. done

Reply:
sorry I didnt understand the **/**etc. I thought you were saying to add comment about high too. my mad

Reply:
no worries, in hindsight i can see how that could be misinterpreted